lib/game.rb

```diff
@@ -0,0 +1,94 @@
+require 'pry'
+class Game
+  include Players
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5], # Middle row
+    [6,7,8], # Bottom row
+
+    [0,3,6], #left horizontal column
+    [1,4,7], #Middle horizontal column
+    [2,5,8], #Right horizontal column
+
+    [0,4,8], #left to bottom right diagonal
+    [2,4,6] #right to bottom left diagonal
+  ]
+  attr_accessor :board,:player_1,:player_2
+
+  def initialize(player_1 = Human.new("X"), player_2 = Human.new("O"),board = Board.new)
+    @player_1 = player_1 if player_1
+    @player_2 = player_2 if player_2
+    @board = board if board
+  end
+
+  def current_player
+    #  if board.turn_count.even?
+    #    player_1
+    #  else
+    #    player_2
+    #  end
+     board.turn_count % 2 == 0 ? player_1 : player_2
```

Comment:
Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`

Reply:
`even?` basically does the same as the modulo `i % 2 == 0`.  If the `turn_count` is `0,2,4...` then it's `player_1`'s turn and vice versa.  `.odd?` does the opposite.  It's to show you how many different ways you can accomplish the same thing in Ruby and it's less likely to cause a bug due to a typo.

Reply:
After @board.turn_count.even? the ? will check player 1 and player 2 for if it's even. Could I add a player 3 right after player 2 to check all 3 players?
@player_1 : @player_2 : @player_3

Reply:
Think of a ternary expression the same as an `if... else...` statement. You and try and string multiple expressions (results) together but it'll only return 1 of 2 sets of data based on the condition being evaluated.

Reply:
Alright, I have submitted all the changes. Thank you for help.